src/server/scripts/EasternKingdoms/Stratholme/instance_stratholme.cpp

```diff
@@ -32,18 +32,17 @@ EndScriptData */
 #include "Map.h"
 #include "Player.h"
 #include "stratholme.h"
-
-enum Misc
-{
-    MAX_ENCOUNTER           = 6
-};
+#include "AreaBoundary.h"
```

Comment:
This goes on top, before #include "Map.h"

Reply:
lul
https://www.youtube.com/watch?v=75p-N9YKqNo

Reply:
kekson